cocos/network/CCDownloader-apple.mm

```diff
@@ -444,6 +451,19 @@ - (void)URLSession:(NSURLSession *)session task :(NSURLSessionTask *)task
             }
         }
     }
+    
+    if (error)
+    {
+        NSData *resumeData = [error.userInfo objectForKey:NSURLSessionDownloadTaskResumeData];
+        if(resumeData) {
```

Comment:
Since the error occurs, the `resumeData` **may be broken** and should not be persisted. So it is better to ignore these data.